### PR TITLE
minor fix in creating path

### PR DESCRIPTION
**Pull request by InnaMel**
added check of Null on empty or wrong path

### CODE REVIEW

File: DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs [hunk 1]
```diff
@@ -71,7 +71,7 @@ public async void SaveToTxt()
         {
             var dateCreation = DateTime.Now.ToShortDateString();
             var setFileSaveName = $"Dublicates_{dateCreation}.txt";
-            var setFilePath = Path.Combine($"{PathWithAppFolder}", "{setFileSaveName}");
+            var setFilePath = Path.Combine($"{PathWithAppFolder}", $"{setFileSaveName}");
             var option = new JsonSerializerOptions { WriteIndented = true };
 
             if (File.Exists(setFilePath))
```
Comment (YuriZakitny) [inline DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs:77]: not needed

File: DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs [hunk 2]
```diff
@@ -71,7 +71,7 @@ public async void SaveToTxt()
         {
             var dateCreation = DateTime.Now.ToShortDateString();
             var setFileSaveName = $"Dublicates_{dateCreation}.txt";
-            var setFilePath = Path.Combine($"{PathWithAppFolder}", "{setFileSaveName}");
+            var setFilePath = Path.Combine($"{PathWithAppFolder}", $"{setFileSaveName}");
```
Comment (YuriZakitny) [inline DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs:74]: just PathWithAppFolder, no need in forrmatting

Comment (YuriZakitny) [inline DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs:74]: same here

**Review by YuriZakitny — COMMENTED** [review]
good

File: DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs (hunk 2 above)
Comment (InnaMel) [reply DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs:74]: Fixed. Tnx

Comment (InnaMel) [reply DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs:74]: Fixed. Tnx

**Pull request merged**